symphony/app/fbcnms-projects/magmalte/app/components/TabbedTable.js

```diff
@@ -13,10 +13,63 @@ import React, {useState} from 'react';
 import Tab from '@material-ui/core/Tab';
 import Table from '@material-ui/core/Table';
 import TableBody from '@material-ui/core/TableBody';
+import Grid from '@material-ui/core/Grid';
 import TableCell from '@material-ui/core/TableCell';
 import TableContainer from '@material-ui/core/TableContainer';
 import TableRow from '@material-ui/core/TableRow';
 import Tabs from '@material-ui/core/Tabs';
+import Link from '@material-ui/core/Link';
+import {MuiThemeProvider} from '@material-ui/core/styles';
+import Text from '../theme/design-system/Text';
+import {colors, typography} from '../theme/default';
+import {makeStyles, withStyles} from '@material-ui/styles';
+
+const useStyles = makeStyles(theme => ({
+  tab: {
+    backgroundColor: colors.primary.white,
+    borderRadius: '4px 4px 0 0',
+    boxShadow: `inset 0 -2px 0 0 ${colors.primary.concrete}`,
+
+    '& + &': {
+      marginLeft: '4px',
+    },
+  },
+  emptyTable: {
+    backgroundColor: colors.primary.white,
+    padding: theme.spacing(4),
+    minHeight: '96px',
+  },
+  emptyTableContent: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    color: colors.primary.comet,
+  },
+  rowTitle: {
+    color: colors.primary.brightGray,
+  },
+  rowText: {
+    color: colors.primary.comet,
+  },
+}));
+
+const MagmaTabs = withStyles({
+  indicator: {
+    backgroundColor: colors.secondary.dodgerBlue,
+  },
+})(Tabs);
+
+const MagmaTab = withStyles({
+  root: {
+    fontFamily: typography.body1.fontFamily,
+    fontWeight: typography.body1.fontWeight,
+    fontSize: typography.body1.fontSize,
+    lineHeight: typography.body1.lineHeight,
+    letterSpacing: typography.body1.letterSpacing,
+    color: colors.primary.brightGray,
+    textTransform: 'none',
+  },
+})(Tab);
```

Comment:
is it standard to overwrite the classes by wrapping it in with one more `withStyles`? do we not have any other way to customize the look of Tabs/Tab?

Reply:
Is there any online reference for how to handle this? Does material talk about it being ok to overwrite their styles this way? I worry that if the name of the style changes, it's going to break the component. These classnames aren't necessarily supposed to be part of the public API of a component. I'd almost prefer to make our own tab component than this (assuming I'm right about classnames being private names that aren't guaranteed to stay unchanged).